pkg/util/computeskus/computeskus.go

```diff
@@ -32,3 +34,46 @@ func HasCapability(sku *mgmtcompute.ResourceSku, capabilityName string) bool {
 
 	return false
 }
+
+// Checks whether given resource SKU is restricted in a given location
```

Comment:
We don't use godoc to generate docs for comments, but [there is a convention](https://go.dev/blog/godoc) that function comments must start with function name.

```suggestion
// IsRestricted checks whether given resource SKU is restricted in a given location
```